src/se/vidstige/jadb/DeviceDetectionListener.java

```diff
@@ -0,0 +1,8 @@
+package se.vidstige.jadb;
+
+import java.util.List;
+
+public interface DeviceDetectionListener {
+    public boolean detect(List<JadbDevice> device);
```

Comment:
listeners commonly have a `on<event>` naming. Here it rather sounds like an action rather than an event ("detect"). Perhaps we can improve the naming here?